php/ext/google/protobuf/BUILD.bazel

```diff
@@ -0,0 +1,70 @@
+
+# Protobuf PHP extension (PECL)
+load("@rules_pkg//:mappings.bzl", "pkg_files", "pkg_filegroup")
+load("//:protobuf_version.bzl", "PROTOBUF_PHP_VERSION")
+load("@rules_pkg//:pkg.bzl", "pkg_tar")
+
+package(default_visibility = ["//php:__subpackages__"])
+
+filegroup(
```

Comment:
I don't think you need to make this a filegroup, can you just use the file directly?

Reply:
done